sites/all/modules/custom/jdt_user_notes/jdt_user_notes.module

```diff
@@ -164,17 +143,19 @@ function jdt_user_notes_form($form, &$form_state, $note_id, $article_nid) {
     '#type' => 'textfield',
     '#title' => t('Title'),
     '#value' => isset($note['note_title']) ? $note['note_title'] : '',
+    '#maxlength' => 254,
   );
 
   $form['note_body'] = array(
     '#type' => 'textarea',
     '#title' => t('Body'),
     '#value' => isset($note['note_text']) ? $note['note_text'] : '',
+    '#attributes' => array('maxlength' => 254),
```

Comment:
Unfortunately, this is only client-side validation. Check out form validation and see how you can go about this to validate on the server-side.